docs/s3.md

```diff
@@ -20,8 +20,18 @@ For our use, we will put results relating to a specific release
 in `releases/<release_name>/<ticket>` Using the internal name like Kearney.
 
 If it isn't specific to a release, put it in `tickets/<ticket>`
- 
-## How to access/use it
+
+## General access
+The bucket is public with static web hosting enabled. We also setup a modified version
+of [S3 Bucket Browser](https://github.com/juvs/s3-bucket-browser) to enable normal browsing. 
+To access it you have to use a URL like this
+
+http://slv-performance-results.s3-website-us-west-2.amazonaws.com/
+
+That or anything that points to a folder will launch you into the Bucket Browser. If your url
```

Comment:
URL and HTML should technically be capitalized (not sure if we care for this type of doc).